bpf-apache/bpf.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 Tigera, Inc. All rights reserved.
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
```

Comment:
Copyright _year_ should only be checked for modified files.  You're only supposed to update it when the file is changed.